types/index.d.ts

```diff
@@ -127,8 +127,18 @@ interface IsValidOptions {
   noUndeclaredFields: boolean;
   errorHook: (path: string[], val: any, type: Type) => void
 }
+
+interface ImportHookParams {
```

Comment:
"Params" doesn't feel right for a hook since the data is provided to (not by) the caller. How about `ImportHookData` or `ImportHookPayload`?